src/components/Header.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import BaseComponent from 'lib/BaseComponent';
+import { Link } from 'react-router';
+
+export default class Header extends BaseComponent {
+  render() {
+    return (
+      <div>
+        <div className="header">
+          <div className="header__search">
+            <input
+              className="header__search__main"
+              type="text"
+              placeholder="Search The Gazelle"
+            />
+            <div className="header__search__tail"></div>
+            <div className="header__search__text">SEARCH</div>
+          </div>
+          <div className="header__title">
+            {/* TODO: change link to proper Gazelle icon uploaded to server*/}
+            <Link to="/" className="header__title__content">
+              <img src="http://thegazelle.s3.amazonaws.com/gazelle/2016/02/header-logo.png" alt="logo" />
```

Comment:
All these hard coded links need to be changed to `//thegazelle.s3` links, or you will get mixed content errors on HTTPS.